modules/microshift-rpm-ostree.adoc

```diff
@@ -5,25 +5,27 @@
 :_content-type: PROCEDURE
 [id="microshift-rpm-ostree-package-system_{context}"]
 = rpm-ostree image and package system
+
 To use the HTTP(S) proxy in rpm-ostree, set the `http_proxy environment` variable for the `rpm-ostreed` service.
 
 .Procedure
-. Add the following setting to the `/etc/systemd/system/rpm-ostreed.service.d/00-proxy.conf` file:
+
+. Add this setting to the `/etc/systemd/system/rpm-ostreed.service.d/00-proxy.conf` file by running the following command:
 +
 [source, terminal]
 ----
-Environment="http_proxy=http://$PROXY_USER:$PROXY_PASSWORD@$PROXY_SERVER:$PROXY_PORT/"
+$ Environment="http_proxy=http://$PROXY_USER:$PROXY_PASSWORD@$PROXY_SERVER:$PROXY_PORT/"
```

Comment:
The "$" sign must be removed from here. It's not a command - it's contents of a file.